### PR TITLE
Add a feedback notice to "i did this" buttons.

### PR DESCRIPTION
**Pull request by SamithDB**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

File: app/views/notes/_responses.html.erb [hunk 1]
```diff
@@ -5,8 +5,8 @@
     <% @responses = Tag.find_nodes_by_type("#{response_type}:#{@node.id}", "note", 200) %>
     <% @unpaginated = true %>
  
-    <p id="<%= response_type %>s"><a class='btn btn-primary btn-lg' href="/post?tags=<%= response_type %>:<%= @node.id %><% if response_type == 'response' %>,hidden:response<% end %><%= ',' + (@tagnames.uniq.delete_if{|x| x.match(":") }).join(',') if @tagnames && @tagnames.length > 0 %>"><%= t('notes._responses.post_' + response_type) %></a>&nbsp;
-      <% if @node.power_tag('activity') %>
+    <p id="<%= response_type %>s"><a class='btn btn-primary btn-lg' href="/post?tags=<%= response_type %>:<%= @node.id %><% if response_type == 'response' %>,hidden:response<% end %><%= ',' + (@tagnames.uniq.delete_if{|x| x.match(":") }).join(',') if @tagnames && @tagnames.length > 0 %>"><%= t('notes._responses.post_' + response_type) %></a> Help out by offering feedback!
+	<% if @node.power_tag('activity') %>
```
Comment (seafr) [inline app/views/notes/_responses.html.erb]: Please fix the indentation here by removing the one extra space.

Comment (seafr) [reply app/views/notes/_responses.html.erb]: Everything else if OK! Almost ready for merging.

**Comment by seafr** [comment]
I've edited the PR title to be more descriptive. It is also helpful to reference the issue fixed by the PR, #2475 in this case. Thank you.

**Comment by jywarren** [comment]
Hi, just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!

**Comment by jywarren** [comment]
Hi, @SamithDB -- if you could adjust just that one space, that would be really helpful -- helps keep our code tidy long-term. If not, we're happy to make that final tweak and merge this. Thank you!

**Comment by jywarren** [comment]
I'm going to go ahead and merge here -- thanks everybody!!!

**Pull request merged**